Automattic/simplenote-android
# Add Debug Credentials

### Fix
Add the two items to the `gradle.properties` file so that the email and password fields will be automatically populated for debug builds.  If the `loginEmail` and `loginPassword` items are set in `gradle.properties`,  they will be inserted into the fields for the login flow.  Otherwise, the fields will be empty.

### Test
0. Clear app data.
1. Tap ***Sign Up*** button.
2. Notice ***Email*** and ***Password*** fields are empty.
3. Tap back arrow in app bar.
4. Tap ***Log In*** button.
5. Tap ***Log In with Email*** button.
6. Notice ***Email*** and ***Password*** fields are populated with `loginEmail`  and `loginPassword` values from `gradle.properties`.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
These changes do not require release notes.
